### PR TITLE
{bio}[foss/2023a] chromVARmotifs v0.2.0 w/ R 4.3.2

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by bedroge** [comment]
Test report by @bedroge
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in total)
interactive2 - Linux Rocky Linux 8.9, x86_64, AMD EPYC-Milan Processor (zen2), Python 3.6.8
See https://gist.github.com/bedroge/fba38c22394097920bbbeff242439b79 for a full test report.

**Comment by bedroge** [comment]
@boegelbot please test @ generoso

**Comment by bedroge** [comment]
Test report by @bedroge
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
interactive2 - Linux Rocky Linux 8.9, x86_64, AMD EPYC-Milan Processor (zen2), Python 3.6.8
See https://gist.github.com/bedroge/4c246f7ed655f99dbe6eb8ec8814f8d5 for a full test report.

File: easybuild/easyconfigs/c/chromVARmotifs/chromVARmotifs-0.2.0-foss-2023a-R-4.3.2.eb [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+source_urls = ['https://github.com/GreenleafLab/chromVARmotifs/archive']
+sources = [{'download_filename': '%s.tar.gz' % local_commit, 'filename': SOURCE_TAR_GZ}]
+checksums = ['6bc0894e1ffc9aa51d7eefce71ca0a586b271e7c6fd60211b179862eb51e4f1b']
+
+sanity_check_paths = {
+    'files': [],
+    'dirs': [name],
+}
+
+modextrapaths = {'R_LIBS_SITE': ''}
```
Comment (bedroge) [inline easybuild/easyconfigs/c/chromVARmotifs/chromVARmotifs-0.2.0-foss-2023a-R-4.3.2.eb:30]: @boegel I'm seeing the following when building this:
```
WARNING: Suppressed adding the following path(s) to $R_LIBS_SITE of the module as they were already added: ['']
```
So I guess this line can be removed?
```suggestion
```

Comment (boegel) [reply easybuild/easyconfigs/c/chromVARmotifs/chromVARmotifs-0.2.0-foss-2023a-R-4.3.2.eb:30]: Yes, this is done by `RPackage` easyblock by default, indeed.

**Comment by boegel** [comment]
@boegelbot please test @ jsc-zen3

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3138.skitty.os - Linux RHEL 8.8, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/boegel/3997dc42b0904aa7accbdd68a608e546 for a full test report.

**Review by bedroge — APPROVED** [review]
lgtm

**Comment by bedroge** [comment]
Test report by @bedroge
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
interactive1 - Linux Rocky Linux 8.9, x86_64, AMD EPYC-Milan Processor (zen2), Python 3.6.8
See https://gist.github.com/bedroge/6fb18aee529960cdcb926ca520a70be4 for a full test report.

**Comment by bedroge** [comment]
Going in, thanks @boegel!

**Pull request merged**